shaders/glsl/renderchunk.fragment

```diff
@@ -487,7 +466,7 @@ void main()
 	
 	
 
-	//fake effect when light bounses from surface if hits it with very small angle
+	// Fake effect when light bounces from surface and hits it with very small angle
 	const float fresnelFactor = 4.0;
```

Comment:
"And" is not quite correct in this case. "When" is better

Reply:
Or change order of events. Because light hits the surface and only then bounces from it if it was hitted in a very sharp angle.

Reply:
How about `Fake effect when light bounces from surface and if it is hit with a very small angle`?

Reply:
It is k, just change events order. First it hits and only then bounces

Reply:
Oh ok